presto-testng-services/src/main/java/io/prestosql/testng/services/LogTestDurationListener.java

```diff
@@ -145,8 +144,7 @@ private static void dumpAllThreads(String message)
     {
         LOG.warn("%s\n\nFull Thread Dump:\n%s", message,
                 Arrays.stream(getThreadMXBean().dumpAllThreads(true, true))
-                        // TODO ThreadInfo.toString truncates stacktrace to java.lang.management.ThreadInfo#MAX_FRAMES
-                        .map(ThreadInfo::toString)
```

Comment:
There is a similar code in `io.prestosql.tests.product.launcher.cli.SuiteRun.Execution#reportSuspectedTimeout`. 
Can you please update it too?

Reply:
For sure and thank you for pointing me to the right location :), I apologize for slow responses as I have been pretty busy with finals this week, so I will hopefully be getting around to this come next week :)